app/assets/images/index.ts

```diff
@@ -11,7 +11,8 @@ export const PLANTS_IMAGES = {
 
 export const BACKGROUND_IMAGES = {
   CALM_DEFAULT_BACKGROUND: require('./backgrounds/calmDefaultBackground.png'),
-  GARDEN_BACKGROUND: require('./backgrounds/gardenBackground.png'),
+  GARDEN_DARK: require('./backgrounds/gardenDarkBackground.png'),
+  GARDEN_LIGHT: require('./backgrounds/gardenLightBackground.png'),
   MENU: require('./backgrounds/menuBackground.png'),
```

Comment:
we have alternativeGardenBackground.png
return gardenBackground.png and pull develop